src/Education.UnitTests/EducationTests.cs

```diff
@@ -19,163 +19,140 @@
  */
 
 using System.Windows.Documents;
-using Moq;
 using SonarLint.VisualStudio.Core;
-using SonarLint.VisualStudio.Core.Suppressions;
 using SonarLint.VisualStudio.Education.Rule;
 using SonarLint.VisualStudio.Education.XamlGenerator;
 using SonarLint.VisualStudio.TestInfrastructure;
 
-namespace SonarLint.VisualStudio.Education.UnitTests
+namespace SonarLint.VisualStudio.Education.UnitTests;
+
+[TestClass]
+public class EducationTests
 {
-    [TestClass]
-    public class EducationTests
+    private readonly SonarCompositeRuleId knownRule = new("repoKey", "ruleKey");
+    private readonly SonarCompositeRuleId unknownRule = new("known", "xxx");
```

Comment:
This is not unknown until the GetRuleImfoAsync(null) is called that sets it up.

Suggestion:
```
    [TestInitialize]
    public void TestInitialize()
    {
        ...
        SetupKnownRule();
        SetupUnknownRule();
        ...
    }

    private void SetupKnownRule() => ruleMetadataProvider.GetRuleInfoAsync(knownRule, Arg.Any<Guid?>()).Returns(ruleInfo);

    private void SetupUnknownRule() => ruleMetadataProvider.GetRuleInfoAsync(unknownRule, Arg.Any<Guid?>()).ReturnsNull();
```